resources/public/api/conf/1.0/endpoints/charitableGiving_retrieve.raml

```diff
@@ -0,0 +1,27 @@
+is:
+  - headers.acceptJson
+  - headers.testScenarioHeader
+  - errors.formatNino
+  - errors.formatTaxYear
+  - errors.ruleTaxYearRangeInvalid
+  - errors.ruleTaxYearNotSupported
+  - errors.clientOrAgentNotAuthorised
```

Comment:
This description does not quite match the spec documentation

Reply:
With this one the description on the tech specs is the updated description of this error and the delete and create endpoints descriptions should/will also be updated to match this if you have a look through the create and delete tech specs. You could create a clientOrAgentNotAuthorised2 or something similar and update the description that way?